src/config.py

```diff
@@ -1,7 +1,7 @@
 app_config = {
     "spark": {
-        "app": "etl-loader",
```

Comment:
Let's make the config file a .json file. For example in `project_root/config/config.json`. Sorry, for changing my mind, but I think it's the best option. Otherwise, the config looks fine.